openwisp_users/apps.py

```diff
@@ -71,3 +82,28 @@ def update_organizations_dict(cls, instance, **kwargs):
         cache.delete(cache_key)
         # forces caching
         user.organizations_dict
+
+    def create_organization_owner(cls, instance, **kwargs):
```

Comment:
you can add a `created` argument and avoid having to use `kwargs.get('created')` (so you can just use `created`) which will make the code easier to read